ui/app/AppLayouts/Communities/panels/PermissionsSettingsPanel.qml

```diff
@@ -198,12 +198,12 @@ StackView {
                     return false
                 }
 
-                permissionTypeLimitReached: {
+                readonly property var permissionTypeLimitReachedOrExceeded: {
```

Comment:
mmm I don't understand this `exceeded` condition.. Can you explain me the case??

On this issue what you should detect, imo, is if you are entering in an "update" mode (so, no sense to check the reached condition), and not in a "new permission" creation.